mxnet/inference/docker/1.6.0/py2/Dockerfile.cpu

```diff
@@ -15,8 +15,7 @@ ARG MX_URL=https://aws-mxnet-pypi.s3-us-west-2.amazonaws.com/1.6.0/aws_mxnet_mkl
 ARG PYTHON=python
 ARG PYTHON_PIP=python-pip
 ARG PIP=pip
-
-ARG OPENSSL_VERSION=1.1.1g
+ARG OPENSSL_VERSION=1.1.1k
```

Comment:
Same here. Could we revert this change. If this was intentional, we could  pull it in as a separate PR.